User Management, VNet CIDR, and API Management Updates (#83)
bcgov/public-cloud-techdocs: docs/azure/design-build-deploy/user-management.md

> As a Product Owner or Technical Lead with restricted Owner permissions, you do not have the ability to:

Reviewer: ```suggestion
As a Product Owner or Technical Lead with restricted Owner permissions, you don't have the ability to:
```

* Assign users to an Owner role
* Assign users to roles at various levels:

   - Project Set Management Group (prefixed with your Project Set license plate, ie. "abc123")
   - Subscription (prefixed with your Project Set license plate, ie. "abc123")
   - Resource groups
   - Individual resources

!!! info "Assigning Users to Roles"

>     The ability to directly assign Users to Roles will be transitioned to using Entra ID Security Groups. This will enable easier auditing of access, in order to comply with regulatory requirements.

Reviewer: ```suggestion
    We are transitioning from direct user role assignments to Entra ID Security Groups. This simplifies access auditing for regulatory compliance.
```

>     You will still be able to directly assign Service Principals to Roles.

Reviewer: ```suggestion
    You'll still be able to directly assign Service Principals to Roles.
```
